src/conv-core/cpuaffinity.C

```diff
@@ -932,10 +932,6 @@ void CmiInitCPUAffinity(char **argv)
   if (pemap != NULL && CmiMyPe()<CmiNumPes()) {    /* work thread */
     int mycore = search_pemap(pemap, CmiMyPeGlobal());
     if(show_affinity_flag) CmiPrintf("Charm++> set PE %d on node %d to core #%d\n", CmiMyPe(), CmiMyNode(), mycore);
-    if (mycore >= CmiNumCores()) {
-      CmiPrintf("Error> Invalid core number %d, only have %d cores (0-%d) on the node. \n", mycore, CmiNumCores(), CmiNumCores()-1);
-      CmiAbort("Invalid core number");
-    }
     if (CmiSetCPUAffinity(mycore) == -1) CmiAbort("set_cpu_affinity abort!");
```

Comment:
Can you add more information to this error message (e.g., "Error setting PE %d on node %d to core #%d")?

Reply:
Done